cfme/infrastructure/provider/__init__.py

```diff
@@ -237,9 +230,9 @@ def hosts(self):
         for host in self.get_yaml_data().get("hosts", []):
             creds = conf.credentials.get(host["credentials"], {})
             cred = Host.Credential(
-                principal=creds["username"],
-                secret=creds["password"],
-                verify_secret=creds["password"],
+                principal=creds["principal"],
```

Comment:
nope, I suppose someone has changed credentials file a lot of time ago but hasn't updated tests.
This was a test fix.